packages/@aws-cdk/aws-ec2/lib/machine-image.ts

```diff
@@ -30,10 +30,8 @@ export interface MachineImageConfig {
 
   /**
    * Initial UserData for this image
-   *
-   * @default - Default UserData appropriate for the osType is created
    */
-  readonly userData?: UserData;
+  readonly userData: UserData;
```

Comment:
This be a breaking change?

Reply:
We don't expect people to instantiate these classes themselves, we expect them to use the existing `MachineImages`. But I'll add a note.